vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddModuleController.java

```diff
@@ -37,4 +42,12 @@ public String addModule(Model model, @ModelAttribute ModuleForm moduleForm, Prin
 		moduleManager.storeModule(module);
 		return "redirect:/staff/module/list";
 	}
+	
+	@RequestMapping(value = "/staff/module/{moduleId}/{sequenceId}", method = RequestMethod.POST)
```

Comment:
since this would set the start sequence, the path should reflect that, e.g.
`/staff/module/{moduleId}/sequence/start/{seqId}`